src/Core/src/Platform/Windows/KeyboardAcceleratorExtensions.cs

```diff
@@ -0,0 +1,137 @@
+﻿using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Windows.System;
+
+namespace Microsoft.Maui.Platform
+{
+	public static class KeyboardAcceleratorExtensions
+	{
+		public static void UpdateAccelerator(this MenuFlyoutItemBase platformView, IMenuFlyoutItem menuFlyoutItem)
+		{
+#if NETCOREAPP || NETSTANDARD2_1_OR_GREATER
+			var keyboardAccelerators = menuFlyoutItem.Accelerator?.ToPlatform();
+#else
+			var keyboardAccelerators = menuFlyoutItem.Accelerator.ToPlatform();
+#endif
+
+			if (keyboardAccelerators is null)
+				return;
+
+			foreach (var keyboardAccelerator in keyboardAccelerators)
+				platformView.KeyboardAccelerators.Add(keyboardAccelerator);
+			
+		}
+
+		// Single key (A, Delete, F2, Spacebar, Esc, Multimedia Key) accelerators and multi-key
+		// accelerators (Ctrl+Shift+M) are supported.
+		// Gamepad virtual keys are not supported.
+		public static IList<KeyboardAccelerator>? ToPlatform(this IAccelerator accelerator)
+		{
+			if (accelerator is null)
+				return null;
+
+			List<KeyboardAccelerator> result = new List<KeyboardAccelerator>();
+			
+			var key = accelerator.Key;
+			var modifiers = accelerator.Modifiers;
+
+			if (modifiers is not null)
+			{
+				var modifiersCount = modifiers.Count();
+				bool hasModifierMask = modifiersCount > 0;
+
+				if (hasModifierMask)
+				{
+					if (modifiersCount == 1)
+						result.Add(CreateSingleKeyAccelerator(modifiers!.ElementAt(0), key));
```

Comment:
I don't think either of these need null forgiveness operators.

Reply:
Already modified.